packages/common/core-interfaces/src/logger.ts

```diff
@@ -32,16 +35,19 @@ export type TelemetryEventPropertyType = string | number | boolean | undefined;
  *
  * This indicates that the value should be organized or handled differently by loggers in various first or third
  * party scenarios. For example, tags are used to mark data that should not be stored in logs for privacy reasons.
+ * @internal
  */
 export interface Tagged<V, T extends string = string> {
 	value: V;
 	tag: T;
 }
 
 /**
+ * TODO: add description
```

Comment:
I think I would prefer adding eslint-ignore comments before each of these, rather than adding TODOs directly into the docs. Otherwise our API reports will claim items are documented that aren't.

Reply:
i can see if i can do that